prometheus-ksonnet/lib/prometheus.libsonnet

```diff
@@ -5,6 +5,7 @@ k {
     prometheus_external_hostname: "http://prometheus.%s.svc.cluster.local" % $._config.namespace,
     prometheus_path: "/",
     prometheus_port: 80,
+    prometheus_web_route_prefix: "/",
```

Comment:
To make this compatible with what we do, could you set this to default to `$._config.prometheus_path`?